doc/Development_Documentation/26_Best_Practice/55_Primary-Replica_Database_Connection.md

```diff
@@ -1,61 +0,0 @@
-
-# How to configure Pimcore to use a Primary/Replica Database Connection 
-**IMPORTANT**: Please be aware that the primary/replica connection can only be used for a clustered MariaDB/MySQL environment, **NOT** 
```

Comment:
This is still relevant information. 
I'd keep that page and for configuration I'd link to the official primary/replica docs of Doctrine Bundle. WDYT?